core/src/services/s3/backend.rs

```diff
@@ -512,6 +512,7 @@ impl S3Builder {
 
         // Update with endpoint templates.
         endpoint = if let Some(template) = ENDPOINT_TEMPLATES.get(endpoint.as_str()) {
+            #[allow(clippy::literal_string_with_formatting_args)]
             template.replace("{region}", region)
```

Comment:
Hi, it's not a formatting_args, `{region}` is the raw content.

Reply:
yes, I checked the code, that's why ignore clippy here

Reply:
more info:
without the ignore
![image](https://github.com/user-attachments/assets/1469b7dd-ab7e-42d2-ad86-7d746a5d95fa)

Reply:
It seems you are not using the stable version of Rust to run Clippy. I believe we don't need to add this at the current stage.

Reply:
In that case, it's a false positive. Perhaps we could raise an issue with Clippy instead. `std::replace()` takes a `Pattern` instead of a `format_args()`, it doesn't make sense to check this here.

Reply:
> It seems you are not using the stable version of Rust to run Clippy. I believe we don't need to add this at the current stage.

OK copy that, will drop this

Reply:
dropped

Reply:
Related to https://github.com/rust-lang/rust-clippy/issues/13989

Reply:
cool